pkg/okteto/kubetoken_client.go

```diff
@@ -46,24 +52,40 @@ func NewKubeTokenClient(contextName, token, namespace string) (*KubeTokenClient,
 	}, nil
 }
 
-func (c *KubeTokenClient) GetKubeToken() (string, error) {
+func (c *KubeTokenClient) RequestKubeToken() (*KubeTokenResponse, error) {
 	resp, err := c.httpClient.Get(c.url)
 	if err != nil {
-		return "", fmt.Errorf("failed GET request: %w", err)
+		return &KubeTokenResponse{}, fmt.Errorf("failed GET request: %w", err)
```

Comment:
The idiomatic way of returning an error would be to return a nil struct or removing the pointer at the func signature:
`func (c *KubeTokenClient) RequestKubeToken() (KubeTokenResponse, error)`